deploy/provision/unifi-probe.ps1

```
@@ -0,0 +1,280 @@
+$GhostCount = @($GhostDevices).Count
+$MatchCount = @($HostMacMatches | Where-Object { $_.status -eq "matched" }).Count
+$TopologyOk = ($GhostCount -eq 0 -and $MatchCount -gt 0)
+
+# --- Helpers to render JSON arrays without ConvertTo-Json depth issues ---
+function To-JsonArray($items, [scriptblock]$itemRenderer) {
+    if (-not $items -or @($items).Count -eq 0) { return "[]" }
+    $parts = @($items) | ForEach-Object { & $itemRenderer $_ }
+    return "[" + ($parts -join ",") + "]"
+}
+
+function Escape-Json([string]$s) {
```

PSScriptAnalyzer (1.25.0)

[warning] The cmdlet 'Escape-Json' uses an unapproved verb.

(PSUseApprovedVerbs)